tests/cursor/cursor-isDead-001.phpt

```diff
@@ -9,22 +9,25 @@ require_once __DIR__ . "/../utils/basic.inc";
 $manager = new MongoDB\Driver\Manager(STANDALONE);
 
 $bulk = new MongoDB\Driver\BulkWrite();
-$bulk->insert(array('_id' => 1));
-$bulk->insert(array('_id' => 2));
-$bulk->insert(array('_id' => 3));
+$bulk->insert(['_id' => 1]);
+$bulk->insert(['_id' => 2]);
+$bulk->insert(['_id' => 3]);
 $manager->executeBulkWrite(NS, $bulk);
 
-$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query(array(), array('batchSize' => 2)));
+$cursor = $manager->executeQuery(NS, new MongoDB\Driver\Query([], ['batchSize' => 2]));
 
 foreach ($cursor as $_) {
     var_dump($cursor->isDead());
 }
 
+var_dump($cursor->isDead());
+
 ?>
 ===DONE===
 <?php exit(0); ?>
 --EXPECT--
 bool(false)
 bool(false)
+bool(false)
```

Comment:
Note: In libmongoc 1.3.x, `mongoc_cursor_is_alive()` would return false once the cursor entered the final batch. Since this behavior was fixed in [CDRIVER-1221](https://jira.mongodb.org/browse/CDRIVER-1221), our expectation needed to change.